scripts/india_lgd/local_government_directory_districts/preprocess.py

```diff
@@ -45,6 +45,17 @@
     },
 }
 
+# On Wikidata for some of the districts they
+# have created a new entity. Since our DCID
+# is based on the WikidataId, we cant use the new ones
+# as it will change the mapping and hence this override.
+
+WIKIDATAID_DCID_OVERRIDE_MAPPING = {
+    "Q15399": "Q28169759",
+    "Q8560886": "Q110067863",
```

Comment:
https://datacommons.org/browser/wikidataId/Q107016021 doesn't exist, but https://datacommons.org/browser/wikidataId/Q8560886 does.  So, is this entry reversed?

Reply:
@ajaits please have a look as well.